entrypoint.sh

```diff
@@ -53,24 +69,24 @@ HEAD_BRANCH=$(echo "$pr_resp" | jq -r .head.ref)
 
 echo "Base branch for PR #$PR_NUMBER is $BASE_BRANCH"
 
-if [[ "$BASE_REPO" != "$HEAD_REPO" ]]; then
-	echo "PRs from forks are not supported at the moment."
-	exit 1
-fi
+USER_TOKEN=${USER_LOGIN}_TOKEN
+COMMITTER_TOKEN=${!USER_TOKEN:-$GITHUB_TOKEN}
+
+git remote set-url origin https://x-access-token:$COMMITTER_TOKEN@github.com/$GITHUB_REPOSITORY.git
+git config --global user.email "$USER_EMAIL"
+git config --global user.name "$USER_NAME"
```

Comment:
I think you should do the user customization here only if `USER_TOKEN` is defined. Otherwise you should fall back to the original logic. What do you think?

Reply:
To be honest I'd like to be able to know who started the rebase...